<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: rgb(172, 196, 215);
border:1px solid #000; 
border-radius:10px; 
font: 75 11pt &quot;Vemana2000&quot;;
color:#000; 
font-weight:500; 
}</string>
  </property>
  <widget class="QPushButton" name="reButton2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>99</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid #000; 
border-radius:10px; 
background-color: rgb(238, 238, 236);
	color: rgb(32, 74, 135);
font-weight:1000; 
}</string>
   </property>
   <property name="text">
    <string>거절</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reButton1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>99</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid #000; 
border-radius:10px; 
background-color: rgb(238, 238, 236);
	color: rgb(32, 74, 135);
font-weight:900; 
}</string>
   </property>
   <property name="text">
    <string>수락</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pothana2000</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Notification</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
